packages/hebao_v1/contracts/modules/core/WalletFactory.sol

```diff
@@ -298,10 +319,6 @@ contract WalletFactory is MetaTxAware
         );
 
         bytes32 signHash = EIP712.hashPacked(DOMAIN_SEPERATOR, encodedRequest);
-
-        bytes32 txAwareHash_ = txAwareHash();
-        require(txAwareHash_ == 0 || txAwareHash_ == signHash, "INVALID_TX_AWARE_HASH");
```

Comment:
I don't think we need to check this. In the worst case where the user would sign a meta tx with a txAwareHash (though that wouldn't make much sense to begin with) the call will fail when replayed any way because a wallet can only be created once.

Reply:
Ok, but please comment on this `require` and add a comment so later we don't forget why we removed this logic.